README.md

```diff
@@ -14,63 +14,155 @@ If you wish to deploy a local explorer, run:
 $ npm run explore-local
 ```
 
+# Installation
+
+Install dependencies using `yarn`
+
+```bash
+$ yarn
+```
```

Comment:
not `yarn install`?

Reply:
yarn guess commands. Will make it explicit however.